api/v1alpha4/openstackcluster_types.go

```diff
@@ -133,6 +134,44 @@ type OpenStackClusterStatus struct {
 	BastionSecurityGroup *SecurityGroup `json:"bastionSecurityGroup,omitempty"`
 
 	Bastion *Instance `json:"bastion,omitempty"`
+	// FailureReason will be set in the event that there is a terminal problem
+	// reconciling the metal3machine and will contain a succinct value suitable
```

Comment:
`metal3machine` should be `OpenStackMachine` everywhere.

Reply:
Thanks. 
Replaced with `OpenStackCluster`. 
The `OpenStackMachine` already contains the fields.